REQUIRE

```diff
@@ -1,4 +1,4 @@
 julia 0.7
-BinDeps
+BinaryProvider
 
 @osx Homebrew
```

Comment:
We can probably remove `Homebrew` now, right?

Reply:
done